package.json

```diff
@@ -27,6 +27,7 @@
     "check": "npm run check:ts && npm run check:scss && npm run check:dependency-licenses",
     "check:ts": "gts lint",
     "check:scss": "stylelint \"src/**/*.scss\"",
+    "add:license": "license-check-and-add add -r 2022",
     "check:dependency-licenses": "./hack/check-licenses.sh",
     "fix": "npm run fix:ts && npm run fix:scss && npm run fix:html",
```

Comment:
Change to `"fix": "npm run fix:ts && npm run fix:scss && npm run fix:html && npm run add:license",`.